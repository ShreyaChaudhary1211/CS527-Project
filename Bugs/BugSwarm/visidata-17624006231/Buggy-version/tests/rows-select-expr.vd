sheet	col	row	longname	input	keystrokes	comment
			open-file	sample_data/StatusPR.csv	o	
StatusPR			select-expr	'Puerto Rico' in Location	z|	
StatusPR			unselect-expr	'number' in Unit	z\	
StatusPR			dup-selected		"	
