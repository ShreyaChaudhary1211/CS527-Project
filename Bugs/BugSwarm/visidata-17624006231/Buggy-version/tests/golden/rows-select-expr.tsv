Month	Day	Resource	Location	Value	Unit	Source
Sep	29	Electricity	Puerto Rico	5	percent	AEE
Sep	29	Telecomunications	Puerto Rico	30.5	percent	JRTC
Oct	1	Electricity	Puerto Rico	5	percent	AEE
Oct	2	Telecomunications	Puerto Rico	40	percent	JRTC
Oct	2	Water	Puerto Rico	47	percent	AAA
Oct	3	Electricity	Puerto Rico	6.89	percent	AEE
Oct	3	Telecomunications	Puerto Rico	40	percent	JRTC
Oct	3	Water	Puerto Rico	45	percent	AAA
Oct	4	Electricity	Puerto Rico	8.6	percent	AEE
Oct	4	Telecomunications	Puerto Rico	43.32	percent	JRTC
Oct	4	Water	Puerto Rico	48.2	percent	AAA
Oct	5	Electricity	Puerto Rico	9.2	percent	AEE
Oct	5	Telecomunications	Puerto Rico	45	percent	JRTC
Oct	5	Water	Puerto Rico	54.2	percent	AAA
Oct	6	Electricity	Puerto Rico	10.7	percent	AEE
Oct	6	Telecomunications	Puerto Rico	42	percent	JRTC
Oct	6	Water	Puerto Rico	55.5	percent	AAA
Oct	7	Electricity	Puerto Rico	11.7	percent	AEE
Oct	7	Telecomunications	Puerto Rico	44	percent	JRTC
Oct	7	Water	Puerto Rico	56.24	percent	AAA
Oct	8	Electricity	Puerto Rico	11.7	percent	AEE
Oct	8	Telecomunications	Puerto Rico	52	percent	JRTC
Oct	8	Water	Puerto Rico	56.87	percent	AAA
Oct	9	Electricity	Puerto Rico	15	percent	AEE
Oct	9	Telecomunications	Puerto Rico	51	percent	JRTC
Oct	9	Water	Puerto Rico	59.5	percent	AAA
Oct	10	Electricity	Puerto Rico	16	percent	AEE
Oct	10	Telecomunications	Puerto Rico	53	percent	JRTC
Oct	10	Water	Puerto Rico	63.75	percent	AAA
Oct	11	Electricity	Puerto Rico	10.6	percent	AEE
Oct	11	Telecomunications	Puerto Rico	53.6	percent	JRTC
Oct	11	Water	Puerto Rico	64.22	percent	AAA
Oct	12	Electricity	Puerto Rico	17	percent	AEE
Oct	12	Telecomunications	Puerto Rico	55	percent	JRTC
Oct	12	Water	Puerto Rico	64	percent	AAA
Oct	13	Electricity	Puerto Rico	9	percent	AEE
Oct	13	Telecomunications	Puerto Rico	56	percent	JRTC
Oct	13	Water	Puerto Rico	63.2	percent	AAA
Oct	14	Electricity	Puerto Rico	14.6	percent	AEE
Oct	14	Telecomunications	Puerto Rico	58	percent	JRTC
Oct	14	Water	Puerto Rico	64	percent	AAA
Oct	15	Electricity	Puerto Rico	15	percent	AEE
Oct	15	Telecomunications	Puerto Rico	59	percent	JRTC
Oct	15	Water	Puerto Rico	69	percent	AAA
Oct	16	Electricity	Puerto Rico	13.7	percent	AEE
Oct	16	Telecomunications	Puerto Rico	59	percent	JRTC
Oct	16	Water	Puerto Rico	72	percent	AAA
Oct	17	Electricity	Puerto Rico	17.7	percent	AEE
Oct	17	Telecomunications	Puerto Rico	59	percent	JRTC
Oct	17	Water	Puerto Rico	65.43	percent	AAA
Oct	18	Electricity	Puerto Rico	19.1	percent	AEE
Oct	18	Telecomunications	Puerto Rico	61	percent	JRTC
Oct	18	Water	Puerto Rico	69.19	percent	AAA
Oct	19	Electricity	Puerto Rico	21.6	percent	AEE
Oct	19	Telecomunications	Puerto Rico	61	percent	JRTC
Oct	19	Water	Puerto Rico	71.58	percent	AAA
Oct	20	Electricity	Puerto Rico	12.33	percent	AEE
Oct	20	Telecomunications	Puerto Rico	62	percent	JRTC
Oct	20	Water	Puerto Rico	69.51	percent	AAA
Oct	21	Electricity	Puerto Rico	16.61	percent	AEE
Oct	21	Telecomunications	Puerto Rico	63	percent	JRTC
Oct	21	Water	Puerto Rico	71.74	percent	AAA
Oct	22	Electricity	Puerto Rico	20.22	percent	AEE
Oct	22	Telecomunications	Puerto Rico	63	percent	JRTC
Oct	22	Water	Puerto Rico	72.77	percent	AAA
Oct	23	Electricity	Puerto Rico	23	percent	AEE
Oct	23	Telecomunications	Puerto Rico	63	percent	JRTC
Oct	23	Water	Puerto Rico	73.65	percent	AAA
Oct	24	Electricity	Puerto Rico	24.4	percent	AEE
Oct	24	Telecomunications	Puerto Rico	65	percent	JRTC
Oct	24	Water	Puerto Rico	74.37	percent	AAA
Oct	25	Electricity	Puerto Rico	25	percent	AEE
Oct	25	Telecomunications	Puerto Rico	65	percent	JRTC
Oct	25	Water	Puerto Rico	75	percent	AAA
Oct	26	Electricity	Puerto Rico	26.2	percent	AEE
Oct	26	Telecomunications	Puerto Rico	66	percent	JRTC
Oct	26	Water	Puerto Rico	78	percent	AAA
Oct	27	Electricity	Puerto Rico	27.6	percent	AEE
Oct	27	Telecomunications	Puerto Rico	67	percent	JRTC
Oct	27	Water	Puerto Rico	77	percent	AAA
Oct	28	Electricity	Puerto Rico	29.7	percent	AEE
Oct	28	Telecomunications	Puerto Rico	69	percent	JRTC
Oct	28	Water	Puerto Rico	80	percent	AAA
Oct	29	Electricity	Puerto Rico	29.7	percent	AEE
Oct	29	Telecomunications	Puerto Rico	69	percent	JRTC
Oct	29	Water	Puerto Rico	79.52	percent	AAA
Oct	30	Electricity	Puerto Rico	30.5	percent	AEE
Oct	30	Telecomunications	Puerto Rico	69	percent	JRTC
Oct	30	Water	Puerto Rico	81	percent	AAA
Oct	31	Electricity	Puerto Rico	33.4	percent	AEE
Oct	31	Telecomunications	Puerto Rico	70	percent	JRTC
Oct	31	Water	Puerto Rico	82	percent	AAA
Nov	1	Electricity	Puerto Rico	37.9	percent	AEE
Nov	1	Telecomunications	Puerto Rico	70	percent	JRTC
Nov	1	Water	Puerto Rico	82	percent	AAA
Nov	2	Electricity	Puerto Rico	36.9	percent	AEE
Nov	2	Telecomunications	Puerto Rico	70	percent	JRTC
Nov	2	Water	Puerto Rico	83.11	percent	AAA
Nov	3	Electricity	Puerto Rico	37.9	percent	AEE
Nov	3	Telecomunications	Puerto Rico	71	percent	JRTC
Nov	3	Water	Puerto Rico	80.38	percent	AAA
Nov	4	Electricity	Puerto Rico	39.2	percent	AEE
Nov	4	Telecomunications	Puerto Rico	71	percent	JRTC
Nov	4	Water	Puerto Rico	80.56	percent	AAA
Nov	5	Electricity	Puerto Rico	41.03	percent	AEE
Nov	5	Telecomunications	Puerto Rico	71	percent	JRTC
Nov	5	Water	Puerto Rico	83.09	percent	AAA
Nov	6	Electricity	Puerto Rico	42.2	percent	AEE
Nov	6	Telecomunications	Puerto Rico	71.5	percent	JRTC
Nov	6	Water	Puerto Rico	79.46	percent	AAA
Nov	7	Electricity	Puerto Rico	42.4	percent	AEE
Nov	7	Telecomunications	Puerto Rico	72	percent	JRTC
Nov	7	Water	Puerto Rico	83.3	percent	AAA
Nov	8	Electricity	Puerto Rico	42.4	percent	AEE
Nov	8	Telecomunications	Puerto Rico	72	percent	JRTC
Nov	8	Water	Puerto Rico	85.38	percent	AAA
Nov	9	Electricity	Puerto Rico	43.2	percent	AEE
Nov	9	Telecomunications	Puerto Rico	73	percent	JRTC
Nov	9	Water	Puerto Rico	87.81	percent	AAA
Nov	10	Electricity	Puerto Rico	31.2	percent	AEE
Nov	10	Telecomunications	Puerto Rico	72	percent	JRTC
Nov	10	Water	Puerto Rico	78.09	percent	AAA
Nov	11	Electricity	Puerto Rico	44.5	percent	AEE
Nov	11	Telecomunications	Puerto Rico	73	percent	JRTC
Nov	11	Water	Puerto Rico	87.82	percent	AAA
Nov	12	Electricity	Puerto Rico	47.8	percent	AEE
Nov	12	Telecomunications	Puerto Rico	73	percent	JRTC
Nov	12	Water	Puerto Rico	89.61	percent	AAA
